diff --git a/OuterBeyond/THTrace.cs b/OuterBeyond/THTrace.cs
index 61127c9..452693e 100644
--- a/OuterBeyond/THTrace.cs
+++ b/OuterBeyond/THTrace.cs
@@ -86,6 +86,7 @@ namespace OuterBeyond
 		private bool mDroneFiring;
 
 		public THTrace(ContentManager content, Vector2 position, THSaveData saveData)
+			: base(content, position, saveData, THHeroType.Indra)
 		{
 			GlitchGhost[] array = new GlitchGhost[3];
 			GlitchGhost glitchGhost = new GlitchGhost
@@ -128,7 +129,6 @@ namespace OuterBeyond
 			AirGlitchGhosts = array2;
 			mGlitchedFollowers = new List<THNPC>();
 			mAutoGrappleAim = true;
-			base._002Ector(content, position, saveData, THHeroType.Indra);
 			AddState(new THTraceHatchState());
 			AddState(new THHeroIdleState());
 			AddState(new THHeroCrouchState());
